README.md

```diff
@@ -452,6 +452,11 @@ See [worker_types README.md](https://github.com/taskcluster/generic-worker/blob/
 
 # Release notes
 
+In v14.1.0 since v14.0.2
+========================
+
+* [Bug 1436195 - Add support for live-logs via websocktunnel](https://bugzil.la/1436195) (note that existing livelog support via stateless DNS still works)
+
```

Comment:
Could you add a sentence or two to highlight the changes to the config to enable websocktunnel? Thanks!